src/workerd/api/node/zlib-util.c++

```diff
@@ -837,6 +836,8 @@ bool ZlibUtil::BrotliCompressionStream<CompressionContext>::initialize(jsg::Lock
     return false;
   }
 
+  auto results = params.template asArrayPtr<uint32_t>();
```

Comment:
Is there a test that actually fails without this change?

Reply:
Not yet; but I'll make sure that it's covered either from the NodeJS test suite or from a custom test in another PR.